client/src/pages/NoPageFound/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Typography, Button, Box } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const NoPageFound = ({ location }) => {
+  const { pathname } = location;
+  return (
+    <Box display='flex' alignItems='center' justifyContent='center'>
+      <Box display='flex' flexDirection='column' alignItems='center'>
+        <Box mb={5}>
+          <Typography color='primary' variant='h5'>
+            404: Page not found. We&apos;re sorry, we couldn&apos;t find
+{' '}
```

Comment:
remove this unnecessary { ' '}

Reply:
I had issues with this. Some sort of conflict with the auto-formatting on save. I'd appreciate any and all help on this. I'll leave it in for this PR?